### ISSUE TITLE
Sprite Nine Slice Setup Improvement

### ISSUE THREAD

**Issue by Sererena** [YoYoGames/GameMaker-Bugs#4405]
### Is your feature request related to a problem?

It's tedious to set up individual nine slice sprites, especially setting each slice individually.  If you have multiple sprites with nine slice, it takes a long time to set them up.

### Describe the solution you'd like

Under Preferences > Sprite Editor > Nine Slice, let the user set a default tile mode for all slices of a newly activated nine slice sprite. (Similar to having a default origin for imported sprites)
This will save the user 5 extra steps (and 10+ clicks) per nine slice sprite.

**Comment by mistletoe** [comment]
Agreed. I use Nine Slice constantly for tiling stuff, and each time, the settings are 2,2,2,2, center Tile, lol.
